pkg/golinters/depguard.go

```diff
@@ -0,0 +1,58 @@
+package golinters
```

Comment:
add a test file to `pkg/testdata` please

Reply:
Found it. I use Fedora and the golang package that I installed strips out those files...